src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -44,10 +54,43 @@ export const TrackPanelBookmarks = (props: TrackPanelBookmarksProps) => {
 
       return (
         <dd key={exampleObject.object_id}>
-          <Link to={path}>
-            {upperFirst(exampleObject.object_type)}:{' '}
-            {getExampleObjLabel(exampleObject)}
+          <Link to={path}>{getExampleObjLabel(exampleObject)}</Link>
+          <span className={styles.previouslyViewedType}>
+            {' '}
+            {upperFirst(exampleObject.object_type)}
+          </span>
+        </dd>
+      );
+    });
+  };
+
+  const getBookmarkedLinks = () => {
```

Comment:
This also looks like a React component (`<PreviouslyViewed items={props.previouslyViewed} />`) rather than a function inside another component.

Reply:
That's done